config/locales/diaspora/en.yml

```diff
@@ -436,22 +436,24 @@ en:
     private_profiles:
```

Comment:
I suggest combining the two profile sections. I'll comment in the main conversation.